### PR TITLE
Add test coverage badge TEMP (need modification once tests work)

### PR DESCRIPTION
**Pull request by reginawang99**
# Summary
- add badge to README.md for code coverage
- code coverage (git action) is triggered on push to master. 
     - we aren't calculating code coverage on every pull request
     - doesn't fail even if go tests fail
- see [coveralls](https://coveralls.io/github/uclaacm/teach-la-go-backend) for coverage details

## Explanation
- considered having a static badge like [this](https://github.com/jpoles1/gopherbadger)
- decided on using coveralls (external service) because it allows for better visibility on the areas we lack tests on

### Explanation for ` continue-on-error: true ` for go test coverage
We should probably remove this line in the future, but for now, it'd be nice to have coverage even if the tests were failing.

# Test
- take a look at https://coveralls.io/github/uclaacm/teach-la-go-backend for the coverage details

### CODE REVIEW

File: .github/workflows/go-coverage.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Calc Coverage
+      run: |
```
Comment (reginawang99) [inline .github/workflows/go-coverage.yml:17]: change to singular line

File: .github/workflows/go-coverage.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14'
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Calc Coverage
+      run: | 
+        go test -v ./... -covermode=count -coverprofile=coverage.out
```
[the remark this answers is not in the document]
Comment (reginawang99) [reply .github/workflows/go-coverage.yml:18]: maybe compile this with the other go tests rather than it being its own file?

**Review by lumisphere902 — CHANGES_REQUESTED** [review]
Hi Regina, thanks for working on this. All of it looks pretty good; I just have one concern. It seems like there’s a failing check because of the coverage? I’d prefer if that wasn’t there, but otherwise LGTM

**Comment by reginawang99** [comment]
# Coveralls Coverage
Coveralls will fail if coverage drops by more than 10%
You can modify this in Coveralls like so
image

# Once tests are fixed
- remove the `test` job
- remove `continue-on-error: true` from `test-coverage`

**Review by krashanoff — APPROVED** [review]
Thanks for the changes Regina. LGTM.

**Pull request merged**